core/server.go

```diff
@@ -647,6 +650,7 @@ func (m *RpcServer) GetWorkflowTemplates(cxt context.Context, req *pb.GetWorkflo
 					Repo: string(repo),
 					Revision: string(revision),
 					Template: template.Name,
+					Description: varSpecMap["description"].DefaultValue,
```

Comment:
I'm not sure this will work. The description is not a variable and should have no associated `VarSpec`. A `VarSpec` is a structure that declares a user-facing variable, but a description is a read-only property of a workflow template (and by extension, of an environment).

Have you tested this?